### ISSUE TITLE
improve flow of group order when a ordergroup has less than minimum balance 

### ISSUE THREAD

**Issue by yksflip** [foodcoops/foodsoft#1032]
It's pretty confusing for users to complete a group order when their balance is less than the configured minimum balance. Currently  all sum table fields turn red and the save button is disabled. It's not understandable what's going on and that you cannot save your order anymore.

I'd suggest:
* If your ordergroup has insufficient funds before starting the order, you are not able to edit a grouporder at all, you'll get a warning and get redirected to your grouporder profile where you can see your balance.
* If you have enough funds to start a order and order more than your balance, you can still click the save button, but a error modal opens with a explanatory message and you can adapt your order again.
  * Another idea would be to save a "draft"-order that are incomplete orders.  But I think this would be a much bigger change and open new design questions ...

**Comment by twothreenine** [comment]
> * If your ordergroup has insufficient funds before starting the order, you are not able to edit a grouporder at all, you'll get a warning and get redirected to your grouporder profile where you can see your balance.

I think you should still be able to view the order and which articles are offered before you decide whether you want to fill up your balance (or by how much). But we could use a warning popup/modal which can be dismissed and/or disable all `+` buttons of articles with a positive gross price.

> * If you have enough funds to start a order and order more than your balance, you can still click the save button, but a error modal opens with a explanatory message and you can adapt your order again.

A simple text "insufficient balance" next to the "save" button as soon as you go under the minimum would solve it pretty well, I guess?

also related to #1040

**Comment by yksflip** [comment]
Yes I understand It would be helpful to actually see what's ordered. 
I don't like the idea of disabling parts of the form very much, but would be ok if we don't find another solution.
I think I'd rather like to present the show view of the grouporder (with a notice)  instead of the edit if there's not enough funds?

**Comment by twothreenine** [comment]
> Yes I understand It would be helpful to actually see what's ordered.

That too, but I meant what's _offered_.

> I think I'd rather like to present the show view of the grouporder (with a notice) instead of the edit if there's not enough funds?

Hmm, so that an empty grouporder would be created and then shown? In the show view of the grouporder articles you haven't ordered are hidden by default, so you would need to find the `Show/hide articles not ordered` button and even then, you couldn't see origin and producer information.
